run_dir/design/flowcells.html

```diff
@@ -13,11 +13,11 @@
   <div id="fc_table_div">
```

Comment:
I saw that the header "Showing flowcells from the last 6 months." look larger now. I think it would look nicer with a smaller one, like the original one.

Reply:
Reduced the size to same as the original.